dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/LambdaEventDAO.java

```diff
@@ -69,7 +69,7 @@ public List<LambdaEvent> findByUser(User user, String offset, Integer limit) {
 
         query.select(event);
 
-        int primitiveOffset = Integer.parseInt(MoreObjects.firstNonNull(offset, "0"));
+        int primitiveOffset = (offset != null) ? offset : 0;
         TypedQuery<LambdaEvent> typedQuery = currentSession().createQuery(query).setFirstResult(primitiveOffset).setMaxResults(limit);
```

Comment:
Nit, and was like this already, but there is a null check for `offset` but not for `limit`.

Also, you could argue it both ways, and I also know it was this way already, so no need to change, but my inclination would be to have line 60 declare primitives, `int` instead of `Integer`, and then force the caller to decide what the defaults should be.